lib/ex_prompt.ex

```diff
@@ -168,4 +190,56 @@ defmodule ExPrompt do
         pw_loop(prompt)
     end
   end
+  
+  defp format_trailing(prompt, default \\ nil) do
+    case Regex.named_captures(~r/(?<spaces>[[:space:]]*)$/, prompt) do
+      %{"spaces" => ""} when default != nil ->
+        "#{String.trim_trailing(prompt)}\s#{default}\s"
```

Comment:
For readability sake, is there something that does not work if we put the "literal" spaces directly in the string instead of using the `\s` form?

Reply:
Basically what I'm trying to get at, can't we do something like:

`"#{String.trim_trailing(prompt) #{default}"`

Reply:
Both ways work fine. The idea was to show how many spaces are in the string explicitly, but I'll replace all `\s` with the literal form.

Reply:
Hey man! Sorry I was referring to the spaces in the file, not those in the tests; as in there I think they do help understand that the characters and how they work 🤔 I can do it myself though 🙏 

I will test the functionality meanwhile during the day 👨🏻‍💻